_posts/2016/01/2016-01-28-instructor-debriefing-round-02.md

```diff
@@ -0,0 +1,55 @@
+1. only a handful of students use them before coming to the workshop
+2. instructors/helpers use them when diagnosing install problems on site
+3. some cleaning of the script would be useful to remove checks for outdated and/or irrelevant tools
+
+If you want to contribute to your two cents, the you can submit an issue [here](https://github.com/swcarpentry/windows-installer).
+
+##Thoughts on dealing with installation problems
+We know that poor planning, sub-par wifi, lack of admin privileges, improper installs, power outages, and a bunch of other things can give rise to installation problems for a learner. 
+
+So, what do you do when this happens? One idea is to have a handful of USB sticks with all the files needed for quick transfer. This has its own issues but is a good plan B. Maneesha Sane and Kate Hertweck are working on a list of things that should be on this. 
+
+##What worked well
+1. The Oklahoma group is developing a Git that they are very happy with. Check it out [here](http://github.com/oulib-swc).
+2. The Boston group gave a 10 minute session on implicit bias and stereotype threat that went over really well. Multiple said they were glad it was included. See [this link for details] (http://pad.software-carpentry.org/swc-debriefing-morning) or check out these [two](http://womeninastronomy.blogspot.com/2014/05/why-we-resist-unconscious-bias.html) [blogs](https://dynamicecology.wordpress.com/2014/04/28/stereotype-threat-a-summary-of-the-problem/). 
+3. Learners really love the [Data Carpentry Spreadsheets Lesson](http://www.datacarpentry.org/spreadsheet-ecology-lesson/). The lesson is still under development and has room for improvment, but it is helping people make the transition from using Excell to using the command line in their research
+
+##What could have gone better
+1. The U Boston group need to cap the number of U Boston students admitted to allow seats for non-U Boston attendees, but this was tricky to do within Eventbright. 
+2. The U Washington group taught two concurrent workshops, with a R-centric group in one room and a Python-centric group in the other. Managing a single waitlist for both on Eventbright was not smooth. Maneesha is looking into solutions for this.
+3. The U Washington instructors taught Bash and Git in the mornings, think its best to teach this in the morning. But, the learning didn't like having their 6 hour R or 6 hour Python lesson split up in two. **What do y'all think? What's the best order for teaching?**
+
+## Thanks!
+We are grateful to the instructors who attended the debriefing sessions this round. By taking the time to share their experience and listen to the experiences of others, they are truly making the Software and Data Carpentry community even more awesome!
+
+Morning attendees
+- Ariel Rokem [University of Washington] (http://uwescience.github.io/2016-01-07-uw) 
```

Comment:
I would like to drop the workshop link. Isn't clear if the universities are the instructors employers or the workshop hosts.